test/src/test/java/org/axonframework/test/aggregate/FixtureTest_Deadlines.java

```diff
@@ -90,13 +93,139 @@ void testExpectNoScheduledDeadlineWithName() {
+        fixture.givenNoPriorActivity()
+               .andGivenCommands(CREATE_COMMAND)
+               .whenThenTimeElapses(Duration.ofMinutes(TRIGGER_DURATION_MINUTES + 1))
+               .expectTriggeredDeadlines("deadlineDetails");
+    }
+
+    @Test
+    void testTriggeredDeadlinesFailsForIncorrectDeadlines() {
+        AxonAssertionError result = assertThrows(
+                AxonAssertionError.class,
+                () -> fixture.givenNoPriorActivity()
+                             .andGivenCommands(CREATE_COMMAND)
+                             .whenThenTimeElapses(Duration.ofMinutes(TRIGGER_DURATION_MINUTES + 1))
+                             .expectTriggeredDeadlines("none-occurring-deadline")
+        );
+
+        assertTrue(
+                result.getMessage().contains("Expected deadlines were not triggered at the given deadline manager.")
+        );
+    }
+
+    @Test
+    void testTriggeredDeadlinesFailsForIncorrectNumberOfDeadlines() {
+        AxonAssertionError result = assertThrows(
+                AxonAssertionError.class,
+                () -> fixture.givenNoPriorActivity()
+                             .andGivenCommands(CREATE_COMMAND)
+                             .whenThenTimeElapses(Duration.ofMinutes(TRIGGER_DURATION_MINUTES + 1))
+                             .expectTriggeredDeadlines("deadlineDetails", "none-occurring-deadline")
+        );
+
+        assertTrue(result.getMessage().contains("Got wrong number of triggered deadlines."));
+    }
+
+    @Test
+    void testTriggeredDeadlinesWithName() {
+        fixture.givenNoPriorActivity()
+               .andGivenCommands(CREATE_COMMAND)
+               .whenThenTimeElapses(Duration.ofMinutes(TRIGGER_DURATION_MINUTES + 1))
+               .expectTriggeredDeadlinesWithName("deadlineName");
+    }
+
+    @Test
+    void testTriggeredDeadlinesWithNameFailsForIncorrectDeadlines() {
+        AxonAssertionError result = assertThrows(
+                AxonAssertionError.class,
+                () -> fixture.givenNoPriorActivity()
+                             .andGivenCommands(CREATE_COMMAND)
+                             .whenThenTimeElapses(Duration.ofMinutes(TRIGGER_DURATION_MINUTES + 1))
+                             .expectTriggeredDeadlinesWithName("none-occurring-deadline")
+        );
+
+        assertTrue(
+                result.getMessage().contains("Expected deadlines were not triggered at the given deadline manager.")
+        );
+    }
+
+    @Test
+    void testTriggeredDeadlinesWithNameFailsForIncorrectNumberOfDeadlines() {
+        AxonAssertionError result = assertThrows(
+                AxonAssertionError.class,
+                () -> fixture.givenNoPriorActivity()
+                             .andGivenCommands(CREATE_COMMAND)
+                             .whenThenTimeElapses(Duration.ofMinutes(TRIGGER_DURATION_MINUTES + 1))
+                             .expectTriggeredDeadlinesWithName("deadlineDetails", "none-occurring-deadline")
```

Comment:
nit: you can make all those Strings as constants

Reply:
Good point. Will do.